DFC.Digital/DFC.Digital.Web.Sitefinity.DfcSearchModule/Extensions/HelperExtensions.cs

```diff
@@ -17,6 +17,13 @@ internal static IEnumerable<JobProfileIndex> ConvertToJobProfileIndex(this IEnum
             List<Task> salaryPopulation = new List<Task>();
             foreach (var item in documents)
             {
+                bool itemIsImported = false;
+                bool result;
+                if (bool.TryParse(item.GetValue(nameof(JobProfileIndex.IsImported))?.ToString(), out result))
```

Comment:
We should use nameof(JobProfile.IsImported) instead. Ideally we should start switching all of the fields to use the JobProfile domain model, as JP index can evolve differently to JP domain model itself, which we are ensuring it to be same as sitefinity dynamic content field.

Reply:
Got the point, but we still need IsImported and we don't populate it in JobProfileIndex. ... And soon is going to be removed ...